requirements.txt

```diff
@@ -5,14 +5,14 @@ cryptography>=3.2
 importlib-metadata>=1.7.0 # (included in Python 3.8 by default.)
 Ipython>=7.16.3
 ipywidgets>=7.5.1
-jinja2>=2.10
+jinja2>=2.11.3
 jsonpatch>=1.22
 jsonschema>=2.5.1
 makefun>=1.7.0,<2
 marshmallow>=3.7.1,<4.0.0
 mistune>=0.8.4
 nbformat>=5.0
-notebook>=6.4.10
+notebook>=6.4.5
```

Comment:
notebook allows for an earlier version of `nbconvert` to be installed, which was causing problems in Composer